targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CPROTO_064_SB/device/COMPONENT_CM4/TFM_partition/flash_layout.h

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
In commit message for "Add TF-M based partition files - CY8CPROTO_064_SB", specify which tag (if available) or git hash you used to get the flash_layout.h and region_defs.h from

Reply:
Fixed